logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -14,60 +14,60 @@
 
 """Logging handler for App Engine Flexible
 
-Logs to the well-known file that the fluentd sidecar container on App Engine
-Flexible is configured to read from and send to Stackdriver Logging.
+Send logs to Stackdriver Logging API.
+"""
 
-See the fluentd configuration here:
+import os
 
-https://github.com/GoogleCloudPlatform/appengine-sidecars-docker/tree/master/fluentd_logger
-"""
+from google.cloud.logging.handlers.handlers import CloudLoggingHandler
+from google.cloud.logging.handlers.handlers import EXCLUDED_LOGGER_DEFAULTS
+from google.cloud.logging.handlers.transports import BackgroundThreadTransport
+from google.cloud.logging.resource import Resource
 
-# This file is largely copied from:
-#  https://github.com/GoogleCloudPlatform/python-compat-runtime/blob/master
-# /appengine-vmruntime/vmruntime/cloud_logging.py
+GAE_PROJECT_ENV = 'GCLOUD_PROJECT'
```

Comment:
make all of these constants private.

(we can always make them public later if needed)